site/content/contribute/more-info/desktop/developer-setup.md

```diff
@@ -42,12 +42,11 @@ Set up your development environment for building, running, and testing the Matte
 
 #### Mattermost Server
 
-To develop with the Desktop App, we recommend that you setup a Mattermost server and webapp instance specifically for this purpose. This lets you customize it as needed in cases where there are specific integration requirements needed for testing.
+To develop with the Desktop App, we recommend that you setup a Mattermost server specifically for this purpose. This lets you customize it as needed in cases where there are specific integration requirements needed for testing.
 
-You can find information on setting up those instances here:
-
-[Server Developer Setup]({{< ref "/contribute/developer-setup" >}})  
-[Webapp Developer Setup]({{< ref "/contribute/developer-setup" >}})
+You can find information on setting that up here:
+ 
+[Developer Setup]({{< ref "/contribute/developer-setup" >}})
```

Comment:
This isn't the area I was reviewing, but I happened to notice this earlier in the week, so I thought I'd fix it